snuba/datasets/configuration/profiles/entities/chunks.yaml

```diff
@@ -0,0 +1,29 @@
+version: v1
```

Comment:
FYI: You no longer need this file! Storages can be queried directly now (but you can keep it if you want)